### PR TITLE
Mixmix/export dev

### PR DESCRIPTION
**Pull request by mixmix**
fixes https://github.com/entropyxyz/sdk/issues/388

### CODE REVIEW

**Comment by mixmix** [comment]
hey @rh0delta I thought this was ready for review.
If your review is "it's not, I think it needs more, moving it to draft" then can you review it and request changes?

**Comment by rh0delta** [comment]
> hey @rh0delta I thought this was ready for review.
> 
> If your review is "it's not, I think it needs more, moving it to draft" then can you review it and request changes?

My apologies @mixmix should've added a comment for context. I've switched this PR to draft as I have some changes to add on my end in order to get the types and tests working on the CLI. I'll push them up once I get the tests working

**Comment by rh0delta** [comment]
@mixmix my updates are ready for review, very minor addition to the package json field explicitly defining the types destination for the keys export. I will review the rest of the PR, but would like your sign-off on those changes

File: package.json [hunk 1]
```diff
@@ -9,17 +9,25 @@
...
+      "require": "./index.cjs",
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
     },
     "./keys": {
       "import": "./keys.mjs",
-      "require": "./keys.cjs"
+      "require": "./keys.cjs",
+      "default": "./dist/keys/index.js",
+      "types": "./dist/keys/index.d.ts"
```
Comment (rh0delta) [inline package.json:27]: these are my updates @mixmix

**Review by rh0delta — APPROVED** [review]
@mixmix 's changes look good to me, but require his sign-off on my changes before merging this PR

**Review by frankiebee — APPROVED** [review]
Thanks!

**Pull request merged**